dat/board.lua

```diff
@@ -454,6 +454,36 @@ You will still have to escort the ship and land with it to perform the repairs a
       end )
 end
 
+local function board_fcthit_apply ( eat )
+   -- Piss off nearby ships
+   board_plt:distress( player.pilot() )
+
+   local fct = board_plt:faction()
+   local loss = math.floor(-fct:hit( -board_fcthit, system.cur(), "board" ))
```

Comment:
`math.floor` is being used with one parameter, so it is a NOOP. Probably should be removed.